### DISCUSSION TITLE
30 min delay before send a resolve message

### DISCUSSION THREAD

**Discussion by NumberNot** [metalmatze/alertmanager-bot#180] [Help/Support]
Hi, the bot works fairly nice, but is has one disapointed feature - it sends resolving with 30 min delay
 e.g. (I've blanked some useless info)duration of huge trouble had been approx 1m but resolving was send after 29m only
Hi, the bot works fairly nice, but is has one disapointed feature - it sends resolving with 30 min delay
 e.g. (I've blanked some useless info)duration of huge trouble had been approx 1m but resolving was send after 29m only

```
🔥 ContainerKilled -------- 🔥
Labels:
----------
Annotations:
----------
Duration: 30 seconds

✅ ContainerKilled -------- ✅
Labels:
-----------
Annotations:
-----------
Duration: 1 minute
Ended: 29 minutes 30 seconds
```
Is it right behavior?

**Comment by metalmatze** [comment]
It probably is the right behavior. It all depends on the underlying queries that the alerts evaluate. If those take 30min to not be above the threshold anymore, then it'll take that time. Once the alert gets resolved the resolved notification should be there usually in less than 60s.
